recipes/sophios/meta.yaml

```diff
@@ -25,7 +25,6 @@ requirements:
     - python >=3.6
     - python-graphviz
     - jsonschema <4.18
-    - toil
```

Comment:
Is it an optional dependency? If not, you should not remove it.

Reply:
I was trying something, will put it back. Here is the PR for toil recipe: https://github.com/conda-forge/staged-recipes/pull/28113